tests/scattering2d/test_jax_scattering2d.py

```diff
@@ -0,0 +1,256 @@
+import os
```

Comment:
So there are a number of tests here that don't pertain directly to Jax. Specifically, they end up just running backend primitives (pad, unpad, modulus, etc.) with NP inputs. I haven't checked, but I would bet they're identical to the NP backend tests. So the question is, what are we testing here? That the Jax library properly passes these through to the underlying NP implementation?

Another point is that we don't have the same set of tests for 1D and 3D, so why have them here?

Reply:
has this been addressed? I can't tell